hack/jenkins/linux_integration_tests_devshell.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
Do you need to add this job to the "set-pending" script?

Reply:
done